yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -73,6 +73,11 @@ private[spark] class Client(
 
   private val isClusterMode = sparkConf.get("spark.submit.deployMode", "client") == "cluster"
 
+  private val launcherServerPort : Int = sparkConf.get("spark.launcher.internal.port", "0").toInt
+  private val launcherServerSecret : String = sparkConf.get("spark.launcher.internal.secret", "")
```

Comment:
is this config viewable anywhere or doe sit get propogated onto driver and executors?  If so we should remove it from the configs being propogated.